src/components/MultiHopTrade/hooks/useGetTradeQuotes.tsx

```diff
@@ -34,18 +37,21 @@ export const useGetTradeQuotes = () => {
   const debouncedTradeQuoteInput = useDebounce(tradeQuoteInput, 500)
   const sellAsset = useAppSelector(selectSellAsset)
   const buyAsset = useAppSelector(selectBuyAsset)
-  const sellAssetAccountId = useAppSelector(selectSellAssetAccountId)
   const receiveAddress = useReceiveAddress()
   const sellAmountCryptoPrecision = useAppSelector(selectSellAmountCryptoPrecision)
 
+  const sellAssetAccountId = useAppSelector(selectSellAssetAccountId)
+  const buyAssetAccountId = useAppSelector(selectBuyAssetAccountId)
+  const isCrossAccountTrade = useMemo(
+    () => sellAssetAccountId !== buyAssetAccountId,
```

Comment:
Do we want to check for BIP44 params' accountNumber here as a safety net?

Reply:
as discussed account id can work here too

Reply:
Akschually that works just as well, disregard me